src/cms/cms.js

```diff
@@ -32,4 +35,58 @@ CMS.registerPreviewStyle('./../components/all.sass');
 
 CMS.registerWidget('relatedProduct', controlComponent);
 
+CMS.registerEditorComponent({
```

Comment:
What is this for?

Reply:
It's a component for the markdown that allows the user to create a block with image and text.